MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1150 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+    
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem.FieldName == "agree" && e.DataFormItem is DataFormCheckBoxItem dataFormCheckBox)
+        {
+            dataFormCheckBox.Color = Colors.Pink;
+        }
+    }
+    
+{% endhighlight %}
+{% endtabs %}
+
+## Switch Editor
+
+In switch editor, [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) is loaded, and DataForm [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor supports bool data type property.
+
+To add Switch editor in DataForm, register the editor as DataFormEditorType.Switch  for the required property using the `RegisterEditor` method.
+
+{% tabs %}
+{% highlight C# %}
+
+       dataForm.RegisterEditor("CellularData", DataFormEditorType.Switch);
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the Switch Editor thumb color
+
+By default switch editor Thumb color is Blue, you can change the thumb color by using `ThumbColor` property in the `DataFormSwitchItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Agree" && e.DataFormItem is DataFormSwitchItem switchItem)
+        {
+            switchItem. ThumbColor = Colors.Pink;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the Switch editor background color
+
+You can change the switch on color by using the `OnColor` property in the DataFormSwitchItem.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Agree" && e.DataFormItem is DataFormSwitchItem switchItem)
+        {
+            switchItem.OnColor = Colors.Black;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox](https://help.syncfusion.com/maui/combobox)  will be loaded.
+
+#### Changing the ItemsSource of combo box
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the `IDataFormSourceProvider`.
+
+#### Using IDataFormSourceProvider
```

Comment:
#### Using IDataFormSourceProvider

use bold text and not #

**Using IDataFormSourceProvider**